src/main/java/com/servicewizard/ServiceWizard.java

```diff
@@ -0,0 +1,126 @@
+
+package com.servicewizard;
+
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+import com.google.common.io.Files;
+import com.servicewizard.config.ServiceWizardConfiguration;
+import com.servicewizard.config.TransformerConfiguration;
+import com.servicewizard.config.TransformerType;
+import com.servicewizard.locator.JerseyResourceLocator;
+import com.servicewizard.model.ServiceModel;
+import com.servicewizard.transformer.AngularServiceTransformer;
+import com.servicewizard.transformer.TemplateTransformer;
+import com.servicewizard.transformer.Transformer;
+
+public class ServiceWizard {
+
+	public static void main(String... args) throws IOException, ParseException {
+		Options options = new Options()
+				.addOption(Option
+						.builder("t")
+						.longOpt("type")
```

Comment:
Really not happy with the name "type" for this, but it makes more sense than anything else I can think of.